sauna/plugins/ext/http_json.py

```diff
@@ -0,0 +1,119 @@
+from sauna.plugins import Plugin, PluginRegister
+from sauna import DependencyError
+import re
+import json
+
+my_plugin = PluginRegister('HTTP-JSON')
+
+
+@my_plugin.plugin()
+class HTTPJSON(Plugin):
```

Comment:
To really benefit from inheritance and/or code reuse, I would have to refactor ```HTTP```, which I tried to avoid to keep the PR simple with only a new plugin. Plus, keeping the plugins independent seems like a good idea.